index.html

```diff
@@ -54,9 +54,84 @@ <h5 >I help bring awesome web experiences to life, if you have a project in mind
         </main>
 
      <!-- Recent Works -->
-     <section></section>
+        <section class="my_work" id="Portfolio">
+            <h3 class="works_tittle">My recent Works</h3>
+            <div class="grid-container">
+                <div class="work_top">
+                    <div class="main_card">
+                    <h3 id="featured">Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul id="tags_featured"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                    <a href="#" class="project-button">See Project</a>
+                    </div>
+                    <img src="images/Img_Placeholder.png" alt="">
+                </div>
+                <div class="work-1">
+                    <div class="card_content">
+                        <h3>Website Portfolio</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore </p>
+                        <ul id="tags"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                        
+                    </div>
```

Comment:
- For some reason, your project cards are not been rendered properly in the project container. Can we kindly fix that ? 

Here is what I could see 👇🏽 

| Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="388" alt="Screenshot 2023-02-18 at 07 34 58" src="https://user-images.githubusercontent.com/99246359/219868678-ab7f1044-4e6b-4f3c-9cff-7af5788547fd.png"> |<img width="305" alt="Screenshot 2023-02-18 at 07 40 01" src="https://user-images.githubusercontent.com/99246359/219868885-678beda0-4357-415b-8cd0-f776035c12d6.png">  |